dev-tools/reconfigurator-cli/tests/output/cmds-example-stdout

```diff
@@ -1590,6 +1641,8 @@ LEDGERED SLED CONFIG
         all disks reconciled successfully
         all datasets reconciled successfully
         all zones reconciled successfully
+    reference measurements:
+        (measurement set is empty)
```

Comment:
What's "reference measurements" here?  Is this the set of measurements that Nexus has told Sled Agent are allowed right now?

I think more generally I'm confused about the difference between these three sections here ("measurements", "measurement manifest", and "reference measurements").